<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>物流配送路径规划系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="CourierName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="placeholderText">
     <string>快递点名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>快递点</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="CourierX">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="placeholderText">
     <string>位置_x</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="CourierY">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="placeholderText">
     <string>位置_y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CourierButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="targetEditName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="placeholderText">
     <string>目的地名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>目的地</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="targetEditAdd">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="targetEditX">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="placeholderText">
     <string>位置_x</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="targetEditY">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="placeholderText">
     <string>位置_y</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="targetEditDelete">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>80</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="MyGraphicView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>991</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="computePath">
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>590</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>生成路径</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicView</class>
   <extends>QGraphicsView</extends>
   <header>mygraphicview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
